messages_appendix.cddl

```diff
@@ -113,8 +113,9 @@ auth-status = {
 
 ; type key 1005
 auth-spake2-handshake = {
-  0: auth-spake2-psk-status ; psk-status
-  1: bytes ; public-value
+  0: auth-initiation-token; token
```

Comment:
Can this be named `initiation-token` to be more explicit?

Reply:
Thanks for the suggestion, done.

Reply:
Hi, @mfoltzgoogle. Can you help merge this? thanks.